### DISCUSSION TITLE
What's the best practice for upgrading from version 1.3.8 to version 3.0.x?

### DISCUSSION THREAD

**Discussion by aflyflyv2** [apache/dolphinscheduler#11138] [Q&A]
What's the best practice for upgrading from version 1.3.8 to version 3.0.x?

**Comment by Tandoy** [comment]
The latest release version is 2.0.6 can be tried. Deploying 3.x in a production environment is not recommended for the time being.As far as I know 3.0 will be released in Q3.Currently, the community has no plans and instructions for upgrading 2.x to 3.x.
